Add CMLL recognition (#21)
rouxers/book: src/en/cmll.md

<script type="text/javascript" src="twistysim.js"></script>
<style type="text/css" rel="stylesheet">
/* modifies the opacity of the cube wireframe */
.ttk-shp-poly {
    stroke-opacity: 0.3;
}
</style>

# CMLL

## Recognition Method: From beginner's 4-tile to all angles

> CMLL recognition is a little more tricky than something like OLL. Not only does one have to pay attention to the orinetation of the corners, one also has to pay attention to their colors. Despite that, an advanced Roux solver is able to recognize the CMLL case within fractions of a second. Here is the progression towards that.

Reviewer: ```suggestion
CMLL recognition is a little more tricky than something like OLL. Not only does one have to pay attention to the orinetation of the corners, one also has to pay attention to their colors. Despite that, an advanced Roux solver is able to recognize CMLL cases within fractions of a second. Here is the progression towards that.
```

### I. Beginner Recognition

> A beginner most likely uses 2-look CMLL, so it is mostly irrelevant for them to pay attention to the colors of the corners. It is adequate to simply get used to recognizing the orientation of the corners.

Reviewer: ```suggestion
Most beginners use 2-look CMLL, so the colors of the corners are mostly irrelevant. As a result, it is adequate to simply get used to recognizing the orientation of the corners.
```
